lms/static/js/edxnotes/collections/notes.js

```diff
@@ -17,30 +27,26 @@ define([
                     sections = {},
                     units = {};
 
-                if (!courseStructure) {
-                    this.each(function (note) {
-                        var chapter = note.get('chapter'),
-                            section = note.get('section'),
-                            unit = note.get('unit');
+                this.each(function (note) {
```

Comment:
This method should not execute when we disable course structure tab. Can you please confirm this.